2024-03-31 23:26:58,104 | INFO | test_login.py: 25|------开始执行测试用例，开始时间为：2024-03-31 23:26:58------
2024-03-31 23:26:58,105 | INFO | mysql.py: 15|数据库查询结果为：{'id': 1, 'web': '急救云', 'key': 'url_api', 'value': 'http://systest.whsaviour.com/platform-admin'}
2024-03-31 23:26:58,105 | INFO | test_login.py: 65|正在执行用例：正常登录
2024-03-31 23:26:58,105 | INFO | mysql.py: 15|数据库查询结果为：{'id': 1, 'web': '急救云', 'key': 'url_api', 'value': 'http://systest.whsaviour.com/platform-admin'}
2024-03-31 23:26:58,171 | INFO | mysql.py: 15|数据库查询结果为：{'id': 1, 'web': '急救云', 'key': 'url_api', 'value': 'http://systest.whsaviour.com/platform-admin'}
2024-03-31 23:26:58,171 | INFO | test_login.py: 65|正在执行用例：密码错误
2024-03-31 23:26:58,171 | INFO | mysql.py: 15|数据库查询结果为：{'id': 1, 'web': '急救云', 'key': 'url_api', 'value': 'http://systest.whsaviour.com/platform-admin'}
2024-03-31 23:26:58,177 | INFO | mysql.py: 15|数据库查询结果为：{'id': 1, 'web': '急救云', 'key': 'url_api', 'value': 'http://systest.whsaviour.com/platform-admin'}
2024-03-31 23:26:58,177 | INFO | test_login.py: 65|正在执行用例：密码为空
2024-03-31 23:26:58,178 | INFO | mysql.py: 15|数据库查询结果为：{'id': 1, 'web': '急救云', 'key': 'url_api', 'value': 'http://systest.whsaviour.com/platform-admin'}
2024-03-31 23:26:58,183 | INFO | test_login.py: 28|------执行测试结束，完成时间为：2024-03-31 23:26:58------
